projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -570,16 +570,16 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     /** @hidden @internal */
     public validate(control: AbstractControl): ValidationErrors {
         const value: DateRange = control.value;
-        if (this.minValue && value && value.start && DatePickerUtil.lessThanMinValue(value.start, this.minValue, false)) {
+        if (this.minValue && value && value.start && DatePickerUtil.lessThanMinValue(value.start, new Date(this.minValue), false)) {
```

Comment:
This can be done in a separate function to get the date out of value, min, max.

Reply:
done, it is a little more complex now